source/index.ts

```diff
@@ -43,15 +43,15 @@ export interface Ow extends Modifiers, Predicates {
 
 	@param predicate - Predicate used in the validator function.
 	*/
-	create<T>(predicate: BasePredicate<T>): (value: T) => void;
+	create<T>(predicate: BasePredicate<T>): (value: T, label?: string) => void;
 
 	/**
 	Create a reusable validator.
 
 	@param label - Label which should be used in error messages.
 	@param predicate - Predicate used in the validator function.
 	*/
-	create<T>(label: string, predicate: BasePredicate<T>): (value: T) => void;
+	create<T>(label: string, predicate: BasePredicate<T>): (value: T, label?: string) => void;
```

Comment:
This already has a `label` parameter

Reply:
Could you see this as a way for the user to change the label after the validator was created?

Reply:
Can you show an example of how you would use this capability?

Reply:
So you want this?

```js
const checkPassword = ow.create('password', ow.string.minLength(6));

checkPassword('foo', 'overriddenLabel');
```

Reply:
I can see that being useful, yes. But needs to be better documented I think. Maybe move `(value: T, label?: string)` to a `type` and add a doc comment for it?

Reply:
I've just added an interface called `ReusableValidator`.